integration_tests/upgrade/test-upgrade-1.2-to-1.3.sh

```diff
@@ -0,0 +1,1660 @@
+#!/bin/bash
+# Copyright 2020 DSR Corporation
```

Comment:
Discussed with Aziz:
It would be better to have a more realistic upgrade test which perform update from 0.12 to 1.2 and then 1.2 to 1.3.
As for test readability, we can still have multiple modules (files) for each version, and the master test will call all the modules one by one.
It's OK to fix it in a separate PR.